inst/htmlwidgets/lib/leaflet/leaflet.css

```diff
@@ -70,7 +71,7 @@
 	height: 1px;
 	}
 .lvml {
-	behavior: url/**/(#default#VML);
+	behavior: url(#default#VML);

```

Comment:
This change needs to be undone; the `/**/` is necessary to prevent Pandoc from choking when Leaflet is used in an R Markdown document.